tsconfig.json

```diff
@@ -64,7 +64,19 @@
       "@math.gl/web-mercator/*": ["modules/web-mercator/src/*"],
       "@math.gl/web-mercator/test/*": ["modules/web-mercator/test/*"],
       "test/*": ["test/*"]
-    }
+    },
+    "plugins": [
```

Comment:
Could this be managed with some "extend": "..." syntax so that we don't need to duplicate this everywhere?